kind/aliases

```diff
@@ -11,6 +11,7 @@ alias kintd="docker run --network kind --rm -it -v $PWD/kind/kubeconfig:/root/.k
 alias kintk="docker run -v $PWD/kind/kubeconfig:/.kube/config --rm --network kind bitnami/kubectl"
 alias kintnet="docker run --rm -it --network kind nicolaka/netshoot bash"
 alias klint="actionlint .github/workflows/main.yml"
+# TODO: doesn't work with /bin/bash:
```

Comment:
# FIXME: the kintprep  alias doesn't work with /bin/bash:

Reply:
```suggestion
# TODO: the `kintprep` alias doesn't work with /bin/bash
```